queries/leo/highlights.scm

```diff
@@ -0,0 +1,262 @@
+(identifier) @variable 
+
+[
+ "assert"
+ "assert_eq"
+ "assert_neq"
+ "block"
+ "console"
+ "const"
+ "constant"
+ "finalize"
+ "in"
+ "let"
+ "mapping"
+ "private"
+ "program"
+ "public"
+ "record"
+ "self"
+ "struct"
+] @keyword
```

Comment:
"constant", "public", and "private" should be `@type.qualifier`.
"self" should be `@variable.builtin`.

Reply:
You forgot to remove them from here.